akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/ProtobufProtocolConversions.scala

```diff
@@ -128,7 +128,11 @@ private[akka] object ProtobufProtocolConversions {
       event: Event,
```

Comment:
this eventToEnvelope is used from both GrpcReadJournal and EventPusherConsumerServiceImpl